Tasks/AzureFunctionOnKubernetesV0/task.json

```diff
@@ -15,7 +15,7 @@
     "version": {
```

Comment:
You'll need to build these tasks as well, to update the `.loc.json` file or you can manually update them yourself

Reply:
@ds-ms not sure what you're asking here - I've updated the task.jsons for the affected tasks.

Reply:
For every task.json there's a task.loc.json which gets updated when you build that task.
You're missing that update